tests/non-pytest/mypy-ignore-tests/test_consents_usage.py

```diff
@@ -11,7 +12,8 @@
 # create some variant types
 xfer_str: str = "urn:globus:auth:scope:transfer.api.globus.org:all"
 strlist: list[str] = [xfer_str]
-scopelist: list[Scope] = Scope.parse(xfer_str)
+scopelist: list[Scope] = ScopeParser.parse(xfer_str)
+scopeobj: Scope = Scope.parse(xfer_str)
```

Comment:
This seems unrelated... 🤔

Reply:
Oh, that fix made it in separately already. Maybe I need a rebase...